Makefile

```diff
@@ -0,0 +1,116 @@
+# Makefile for Sphinx documentation
+#
+
+# You can set these variables from the command line.
+PYTHON        = python3
+BUILDDIR      = _build
+SPHINXOPTS    =
+SPHINXBUILD   = sphinx-build
+SPHINXLINT    = sphinx-lint
+PAPER         =
+
+# Internal variables.
+PAPEROPT_a4     = -D latex_paper_size=a4
+PAPEROPT_letter = -D latex_paper_size=letter
+ALLSPHINXOPTS   = -d $(BUILDDIR)/doctrees $(PAPEROPT_$(PAPER)) $(SPHINXOPTS) .
+
+.PHONY: help clean html dirhtml singlehtml pickle json  \
+        devhelp epub latex latexpdf text man changes linkcheck doctest htmlview
+
+help:
+	@echo "Please use \`make <target>' where <target> is one of"
+	@echo "  install    to install depedencies"
+	@echo "  html       to make standalone HTML files"
+	@echo "  htmlview   to open the index page built by the html target in your browser"
+	@echo "  clean      to remove the build files"
+	@echo "  dirhtml    to make HTML files named index.html in directories"
+	@echo "  singlehtml to make a single large HTML file"
+	@echo "  pickle     to make pickle files"
+	@echo "  json       to make JSON files"
+	@echo "  epub       to make an epub"
+	@echo "  latex      to make LaTeX files, you can set PAPER=a4 or PAPER=letter"
+	@echo "  latexpdf   to make LaTeX files and run them through pdflatex"
+	@echo "  text       to make text files"
+	@echo "  man        to make manual pages"
+	@echo "  changes    to make an overview of all changed/added/deprecated items"
+	@echo "  linkcheck  to check all external links for integrity"
+	@echo "  doctest    to run all doctests embedded in the documentation (if enabled)"
+
+clean:
+	-rm -rf $(BUILDDIR)/*
+
+install:
+		$(PYTHON) -m pip install -r requirements.txt; \
+		echo "Requirements installed"; \
+	fi
```

Comment:
It looks like there is a stray `fi` statement here from the previous `if` statement that we should probably remove.

Reply:
Fixed, also accidentally removed the `html` make target.